### ISSUE TITLE
Add "Go Back" button when going from Integration detail to Connection

### ISSUE THREAD

**Issue by mastepan** [syndesisio/syndesis#633]
![integrationdetail](https://user-images.githubusercontent.com/8707251/33667199-88d6bbf2-da9c-11e7-8647-4a81977b181e.png)
![backtointegrationdetail](https://user-images.githubusercontent.com/8707251/33667207-8aeef364-da9c-11e7-9cb6-d16b34acd718.png)

It could be a bit complicated because the button should be there only when going from Integration detail page. Is it feasible? The current going from a Connection used in an Integration back is cumbersome.

**Comment by mastepan** [comment]
To be clear I don't propose to add the button exactly where I put it at the screenshot. It is about adding it somewhere where it fits...

**Comment by deeleman** [comment]
> It could be a bit complicated because the button should be there only when going from Integration detail page. Is it feasible?

Any routed component can be triggered from different route paths, so, in short, yes. We just need to use a different route path (either a fragment or a query string parameter) to signal the target Connection details Component that the user is coming from the Integration detail page, and thus the BACK button is required. Such back button would invoke the native JavaScript `window.history.back()` method aaaaaaand.... profit!

In summary it'll be a 5 minutes task.

**Comment by amysueg** [comment]
UX would like to take this on as a sprint work item for the next sprint. UX needs some time to consider this navigation and investigate alternatives.

**Comment by gashcrumb** [comment]
Any word on this one?  If we're going to eventually update the connection detail page does it even make sense to keep this one open?

**Comment by heiko-braun** [comment]
Closing as out-of-date
